style/fonts.css

```diff
@@ -23,3 +23,9 @@
   src: url("./GothamPro/GothamPro-Light.woff2") format("woff2");
   src: url("./GothamPro/GothamPro-Light.woff") format("woff");
 }
+
+@font-face {
+  font-family: Gotham Pro Regular;
+  src: url("./GothamPro/GothamProRegular.woff2") format("woff2");
+  src: url("./GothamPro/GothamProRegular.woff") format("woff");
+} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)